unstructured_inference/inference/layoutelement.py

```diff
@@ -59,7 +59,9 @@ def from_region(cls, region: Rectangle):
         x1, y1, x2, y2 = region.x1, region.y1, region.x2, region.y2
         text = region.text if hasattr(region, "text") else None
         type = region.type if hasattr(region, "type") else None
-        return cls(x1, y1, x2, y2, text, type)
+        prob = region.prob if hasattr(region, "prob") else None
```

Comment:
```suggestion
        prob = getattr(region, "prob")
```
and similar below

Reply:
Actually, after the refactor, this never happens, because Rectangle doesn't contain those properties. Nice catch anyway 😄